fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/SearchQueryRenderer.java

```diff
@@ -710,6 +831,26 @@ public QueryData addSorting(QueryData queryData, String lrAlias) {
         return queryData;
     }
 
+    @Override
+    public QueryData addWholeSystemSorting(QueryData queryData, List<DomainSortParameter> sortParms, String lrAlias) {
+        if (sortParms == null || sortParms.isEmpty()) {
+            return addSorting(queryData, lrAlias);
+        } else {
+            for (DomainSortParameter sortParm : sortParms) {
+                // for whole-system searches, code has been validated to only be _id or _lastUpdated
```

Comment:
is it not possible for a tenant to define their own whole-system search param of type code?
we should be up front about whether this is supported or not.

Reply:
The code referred to here is actually just the sort parameter name. For whole-system search we only support sort parameters of _id and _lastUpdated. As we've since discussed, we will allow user-defined whole-system search parameters of types for which we have global values tables.